### ISSUE TITLE
The current state of this website is appauling. Please read.

### ISSUE THREAD

**Issue by sixteenjune** [ThirdLena/Programming-CAT-2-Website#4]
This website "Programming-CAT-2-Website" Is appauling. Please read the following complaints:

- User experience is lacking.
- Lack of accessibility feautures.
- No light/dark mode toggle.
- Pirated content blatantly in the repository.
- Unoptimised, legacy frameworks (straight HTML, CSS, Lack of Javascript nor Typescript.
- Unmaintained for years, some content may not be releveant.

Thank you for your understanding.

**Comment by ThirdLena** [comment]
I hope u kys (Keep yourself safe)😊

**Comment by sixteenjune** [comment]
Hi

**Comment by ThirdLena** [comment]
bro go away
